### PR TITLE
Install osg-ca-generator from source (SOFTWARE-4680)

### PR DESCRIPTION
**Pull request by brianhlin**

### CODE REVIEW

File: Dockerfile.testing [hunk 1]
```diff
@@ -7,12 +7,18 @@ LABEL maintainer OSG Software <support@opensciencegrid.org>
 
 RUN \
     yum update -y && \
-    yum install -y --enablerepo=devops-itb osg-ca-generator && \
+    yum install -y git && \
```
Comment (matyasselmeci) [inline Dockerfile.testing]:
```suggestion
    (yum install -y git-core || yum install -y git) && \
```
git-core is smaller but it's EL8 only.

File: Dockerfile.testing [hunk 2]
```diff
@@ -7,12 +7,18 @@ LABEL maintainer OSG Software <support@opensciencegrid.org>
...
     yum install -y --enablerepo=osg-upcoming condor && \
     yum install -y python3-pip httpd mod_auth_openidc mod_ssl python3-mod_wsgi && \
-    osg-ca-generator --host && \
     yum clean all && rm -rf /var/cache/yum/*
 
+RUN \
+    git clone https://github.com/opensciencegrid/osg-ca-generator.git && \
```
Comment (matyasselmeci) [inline Dockerfile.testing:17]: Might want to do a `cd /tmp` or something first

**Review by matyasselmeci — APPROVED** [review]
2 suggestions; pre-approving.

File: Dockerfile.testing (hunk 2 above)
Comment (brianhlin) [reply Dockerfile.testing:17]: Done

**Pull request merged**